packages/xstate-react/src/useService.ts

```diff
@@ -1,39 +1,30 @@
 import { useMemo } from 'react';
-import { EventObject, State, Interpreter, Typestate } from 'xstate';
-import { useSubscription, Subscription } from 'use-subscription';
+import { EventObject, State, Interpreter, Typestate, Sender } from 'xstate';
+import { useActor } from './useActor';
+import { ActorRef } from './types';
+
+export function fromService<TContext, TEvent extends EventObject>(
+  service: Interpreter<TContext, any, TEvent, any>
+): ActorRef<TEvent, State<TContext, TEvent>> {
+  const { machine } = service;
+  return {
+    send: service.send.bind(service),
+    subscribe: service.subscribe.bind(service),
+    stop: service.stop,
+    current: service.state || machine.initialState,
```

Comment:
This logs a warning:
https://github.com/davidkpiano/xstate/blob/1273d72447af45b6c92da9b002f82619f566ebb7/packages/core/src/interpreter.ts#L236-L242

You have to check the status of a service before reading this.